docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -1,19 +1,22 @@
 ## Developing a web viewer
 
-### Setup
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
-- Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
+### Develop
+- [Install necessary prerequisites]($docs/getting-started/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @bentley/itwin-viewer --scripts-version @bentley/react-scripts`.

```

Comment:
@kckst8 does anyone already have `cra-template-itwin-viewer` as a package?  We could probably publish both with the @bentley scope and without it...

Reply:
https://github.com/imodeljs/itwin-viewer/tree/master/packages/modules/cra-template-itwin-viewer

Reply:
It's open. What are your thoughts behind multiple names? Not sure it's worth the effort and could make usage tracking a bit more difficult, but maybe I'm missing something?

Reply:
Not sure it's worth the extra work but it would just to not require as much typing :)  so not that important since most people will copy & paste.